src/main/java/hellos/Hellos.java

```diff
@@ -9,6 +9,7 @@ public class Hellos {
   public String generateOutput() {
     StringBuilder builder = new StringBuilder();
 
+    builder.append(ethanSaysHello());
     builder.append(andreasSaysHello());
     builder.append(elySaysHello());
```

Comment:
You've got your alphabetization wrong here, which is why your tests are failing. You want to be after Ely.

```suggestion
    builder.append(andreasSaysHello());
    builder.append(elySaysHello());
    builder.append(ethanSaysHello());
```